src/containers/ToDo.js

```diff
@@ -0,0 +1,133 @@
+import { connect } from 'react-redux';
+import AddListForm from './AddListForm';
+import React, { Component } from 'react';
+import ToDoList from '../components/ToDoList/ToDoList';
+import AddToList from '../components/buttons/AddToList';
+import ResetList from '../components/buttons/ResetList';
+import * as actions from '../redux/actions';
```

Comment:
while importing, import library api at one group and components at another group to maintain clean coding